server.js

```diff
@@ -98,8 +98,8 @@ var config = configure()
 
 var server = require('webserver').create();
 var urlPrefix = config['url'];
-var port = config['port'] || system.env.PORT || 8082;
-var key = config['key'];
+var port = system.env.PORT || config['port'] || 8082;
+var key =  system.env.KEY || config['key'];
 var userAgent = config['user_agent'];
```

Comment:
I don't think that that is something we would change often, but do want to change outside of source-code. For instance, we could do something like "GoAbout/Prerenderer v1.0" and move to "GoAbout/Prerenderer v1.1"... But I don't think we'd want to set different values across Heroku machines?

The KEY thing is something I don't want to end up in a public place.